hypermap/aggregator/models.py

```diff
@@ -356,9 +372,11 @@ def update_thumbnail(self):
             print 'Thumbnail updated for layer %s' % self.name
 
     def worldmap_date_miner(self):
-        year = re.search('\d{2,4} ?B?CE', str(self.title))
+        self.title = str(self.title.encode("ascii", "ignore"))
```

Comment:
this is due to characters that are in other languages no english. i.e strings such as "Canada et Louisiane / par le Sr. le Rouge ingénieur géographe du Roy." or German terms with characters such as umlauts cause the re.search to bring up an error. Using unicode raised the following error for such terms. 
*** UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 45: ordinal not in range(128)
Ascii format removes any of these special language characters. Utf-8 still has them. Please advise